alacritty/res/text.f.glsl

```diff
@@ -32,7 +31,7 @@ void main()
         alphaMask = vec4(1.0);
         color = vec4(bg.rgb, 1.0);
     } else {
-        if (multicolor != 0) {
+        if (fg.a != 0.) {
```

Comment:
We don't trim fraction in shaders, so it should be `0.0`.

Reply:
Sure.